tooling/csv-to-questions.py

```diff
@@ -43,11 +43,12 @@ def main():
 
         for question in questions:
             question_text = question[0].decode("utf-8").replace("\"", "&quot;")  # ...HTML is allowed here.
+            question_score = 0 if question_text.startswith("Test:") else 1
```

Comment:
Not sure I agree with this. However, it should at least print something alerting the user that a question was set to 0 points automatically.

Reply:
I wasn't sure how to go about this, to be honest. I just went with the easiest solution for my use case.

I'm happy to accept suggestions, or revert this part